src/components/JazzIcon/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { useEffect, useRef } from 'react';
+import md5 from 'md5';
+// @ts-ignore
+import jazzicon from '@metamask/jazzicon';
```

Comment:
It's a good idea for a component. Lets add it to the [future-hopr-lib-components](https://github.com/hoprnet/hopr-admin/tree/develop/src/future-hopr-lib-components) folder